src/Message/AbstractMessage.php

```diff
@@ -117,6 +117,11 @@ abstract class AbstractMessage extends AbstractPart
         if (0 === \preg_match('/\d\d:\d\d:\d\d.* [\+\-]\d\d:?\d\d/', $alteredValue)) {
             $alteredValue .= ' +0000';
         }
+        // Handle numeric months
+        $alteredValue = (string) \preg_replace('/^(\d\d) (\d\d) (\d\d(?:\d\d)?) /', '$3-$2-$1 ', $alteredValue);
+        // Handle comments as specified in RFC2822
+        $alteredValue = (string) \preg_replace('/^\s*\(.*?\)\s*/', '', $alteredValue);
```

Comment:
I'm scared: RFC [tells](https://tools.ietf.org/html/rfc2822#appendix-A.5):

1. `(A wonderful \) chap)`
1. `((((()))))`
1. `(a\)) [valid date] (b\))`

are all valid comments. But only the last one is really problematic, because multiple occurrences of comments involve greedy/ungreedy regex aware flag.

I would like to drop support for multiple comments to ease our life, so please:

1. edit the regex in line `115` removing the ` +` part, resulting in `'/\(.*\)/'`
1. remove these 2 extra lines

Reply:
Pushed 👍 

... le email sono terribili 🥶